tasks/system_probe.py

```diff
@@ -1598,8 +1599,12 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
     if windows:
         return
 
+    # only download images not present in preprepared vm disk
+    resp = requests.get('https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/docker.ls')
```

Comment:
Do we need versioning on this file? Otherwise in-progress updates will affect everyone.

Reply:
I think S3 updates are atomic so the file should never be in an invalid state.

Reply:
I meant how do we control updates of the `docker.ls` file, so `ami-builder` branch versions of  of it don't affect everyone/overwrite each other? How we can test changes to `docker.ls` without affecting everyone?

Reply:
Also, how did this file get on S3? I'm not seeing the upload

Reply:
Uploaded here: https://github.com/DataDog/ami-builder/blob/master/.gitlab-ci.yml#L123

As to the versioning question, I think it is a general problem of all artifacts created by ami-builder. I think we should address this problem first in that repository.